src/patterns/definitions.owl

```diff
@@ -69394,8 +69438,8 @@ EquivalentClasses(<http://purl.obolibrary.org/obo/OBA_2041595> ObjectIntersectio
 
 # Class: <http://purl.obolibrary.org/obo/OBA_2041596> (level of flavin-containing monooxygenase 3 in blood serum)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute_location"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/OBA_2041596> "The amount of a flavin-containing monooxygenase 3 when measured in blood serum."^^xsd:string)
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute_location"^^xsd:string) <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/OBA_2041596> "blood serum flavin-containing monooxygenase 3 amount"^^xsd:string)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute_location"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/OBA_2041596> "The amount of a dimethylaniline monooxygenase [N-oxide-forming] 3 when measured in blood serum."^^xsd:string)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "AUTO:patterns/patterns/entity_attribute_location"^^xsd:string) <http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/OBA_2041596> "blood serum dimethylaniline monooxygenase [N-oxide-forming] 3 amount"^^xsd:string)
```

Comment:
Seems like a bigger label change - is this intended?

Reply:
Again, one of the automatic pipelines replaced the term label and the PR name in the text definition with [one of the synonyms of PR:000007576](https://www.ebi.ac.uk/ols4/ontologies/pr/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPR_000007576).
My data table entry has not changed:
```
defined_class	defined_class_name	entity	entity_name	attribute	attribute_name	location	location_name	exact_synonyms	xrefs
OBA:2041596	level of flavin-containing monooxygenase 3 in blood serum	PR:000007576	flavin-containing monooxygenase 3	PATO:0000070	amount	UBERON:0001977	blood serum
```